projects/eval-adxl355-pmdz/src/platform/stm32/platform_src.mk

```diff
@@ -1,14 +1,14 @@
 INCS += $(PLATFORM_DRIVERS)/stm32_delay.h     \
-		$(PLATFORM_DRIVERS)/stm32_gpio.h      \
```

Comment:
as far as i can see in other projects, the *.mk files are using tabs for indentation. so not sure if this change is required.